examples/hono/package.json

```
@@ -1,12 +1,12 @@
 {
-  "version": "1.0.1",
+  "version": "1.0.2",
   "name": "@bun-examples/hono",
   "main": "src/index.js",
   "devDependencies": {
     "bun-types": "^0.0.83"
```

Can you set `bun-types` to latest?

```suggestion
    "bun-types": "latest"
```

---

👍🏻